app/addons/documents/index-results/components/pagination/TableControls.js

```diff
@@ -0,0 +1,64 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
```

Comment:
I made this same comment to Robert back in the day when he did the Redux refactor for the permissions section.  He mentioned the style pattern in the JS community was shifting to uppercase for components.  I could go either way.  Feel strongly I switch it to lowercase?